index.html

```diff
@@ -627,7 +627,7 @@ <h3 class="heading settled dfn-paneled" data-dfn-type="dfn" data-level="4.2" dat
    </ol>
    <h2 class="heading settled" data-level="5" id="priv-sec"><span class="secno">5. </span><span class="content">Security &amp; Privacy Considerations</span><a class="self-link" href="#priv-sec"></a></h2>
    <p>Long Tasks API adheres to cross-origin policy by including origin-safe attribution information about the source of the long task.
-There is a 50ms threshold for long tasks, also timing info provided is clamped to 1ms resolution. Together this provides adequate protection against security attacks against browser.</p>
```

Comment:
Best to update the .bs file (Bikeshed) and then copy over the generated html.
Ideally the hosted HTML is supposed to auto-update with deploy script, but that is broken at the moment.
Bikeshed documentation:
https://tabatkins.github.io/bikeshed/

If you don't want to deal with Bikeshed yet, I am happy to make the change (but good exercise for the longer term)